BootCampHomework1/BootCampHomework1/Program.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+
+namespace BootCampHomework1
+{
+    class Program
+    {
+        /*
+         * Read name, surname, age, weight (in kg) and height(in cm) from console.
+         * Print a message to the screen based on the information given in 1. (an example message shown below):
+         *
+         *    Tom Jefferson is 19 years old, his weight is 50 kg and his height is 156.5 cm.
+         *
+         * Calculate and print body-mass index(BMI)
+         * Do 1 and 2 for another person.         
+		 *
+		 * Making a random change to test what happens :)
+         */
+        static void Main(string[] args)
+        {
+
+            Console.Write("Your name - ");
+            string name = Console.ReadLine() ;
+
+            Console.Write("Your surname - ");
+            string surname = Console.ReadLine();
+
+            Console.Write("Your age - ");
+            int age = int.Parse(Console.ReadLine());
+
+            Console.Write("Your weight in kg - ");
+            string weightString = Console.ReadLine();
+            double weightInKg = double.Parse(weightString);
+
+            Console.Write("Your height in cm - ");
+            string heightString = Console.ReadLine(); 
+            double heightInCm = double.Parse(heightString);
+
+            PrintInfoForPerson(name,surname,age,weightInKg,heightInCm);
```

Comment:
INconsistent functions. If you plan to split small things into functions, it should be done for other things as well:
- Asking and reading user input
- Getting double
- Getting int
- The whole thing for 1 person